cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/meta/RemoteNamespaceQueryHandler.java

```diff
@@ -35,24 +35,24 @@
 @Path(AbstractRemoteSystemOpsHandler.VERSION + "/execute")
 public class RemoteNamespaceQueryHandler extends AbstractRemoteSystemOpsHandler {
```

Comment:
I think we should remove this handler, and change the client to use the handler in master instead.